src/index.js

```diff
@@ -3,11 +3,14 @@ import ReactDOM from 'react-dom';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
 import GlobalStyles from './GlobalStyles.js';
+import { BrowserRouter } from 'react-router-dom';
 
 ReactDOM.render(
   <React.StrictMode>
-    <GlobalStyles />
-    <App />
+    <BrowserRouter>
+      <GlobalStyles />
+      <App />
+    </BrowserRouter>
```

Comment:
just a friendly reminder: If you use react-router and storybook, you have to update your file `preview.js` in the folder `.storybook` like this:
```js
export const decorators = [
  (Story) => (
    <MemoryRouter>
    <GlobalStyles />
    <Story />
    </MemoryRouter>
  )]
```